ios/.gitattributes

```diff
@@ -0,0 +1,4 @@
+# localizable strings for crowdin
+*.strings diff=localizablestrings
+[diff "localizablestrings"]
+    textconv = "iconv -f utf-16 -t utf-8"
```

Comment:
I'll likely revert this file. Was trying to configure so GH would diff the files, but no joy